_pytest/runner.py

```diff
@@ -382,7 +385,8 @@ def __init__(self, longrepr, **extra):
 def pytest_make_collect_report(collector):
     call = CallInfo(
         lambda: list(collector.collect()),
-        'collect')
+        'collect',
+        None)
```

Comment:
We should treat this option as a `bool`, so this should be `False`; but I think we also want to stop on the debugger during collection, so we should pass the parameter here based on `config.getvalue("usepdb")`.

Reply:
what is someone going to do with a debugger in the middle of runner.py?
I guess they could check the sys.path or something like that, but I'm worried about throwing them into a debugger in the middle of the pytest internals